app/middlewares/errors.js

```diff
@@ -5,7 +5,9 @@ const DEFAULT_STATUS_CODE = 500;
 
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
-  [errors.DEFAULT_ERROR]: 500
+  [errors.DEFAULT_ERROR]: 500,
+  [errors.EXTERNAL_API_ERROR]: 502,
```

Comment:
502 use to be a `Bad gateway` error. In this case, I think it would be better a 503 error to the client consuming our api. What do you think?